src/ActiveLogin.Identity.Swedish.TestData/TestDataCSharp.fs

```diff
@@ -0,0 +1,47 @@
+namespace ActiveLogin.Identity.Swedish.TestData
+open ActiveLogin.Identity.Swedish.FSharp
+open ActiveLogin.Identity.Swedish.FSharp.TestData.SwedishPersonalIdentityNumberTestData
+open ActiveLogin.Identity.Swedish
+
+/// A class that provides easy access to the official test numbers for Swedish Personal Identity Number (Personnummer) 
+/// from Skatteverket
+[<CompiledName("SwedishPersonalIdentityNumberTestData")>]
+type SwedishPersonalIdentityNumberTestDataCSharp() =
+    static let toCSharpPin (pin: SwedishPersonalIdentityNumber) =
```

Comment:
What is the accesibility of this, is it public? Does it need to be?

Reply:
I'll gladly take this opportunity to teach some F# 😀

Since this type has a constructor (the empty parenthesis) it is actually a **class** and not a **record** (which it would be without the constructor).

In an F# class all **let** bindings are private and cannot be made to have any other accessibility, they are sort of like private fields. So this function is, and should be, private.

The members in a F# class, indicated by the `member` keyword can have any access modifier but they are public by default.

Reply:
AH, I see, thank you for the good explanation!